plyer/facades.py

```diff
@@ -82,23 +84,61 @@ def take_video(self, filename, on_complete):
         :type filename: str
         :type on_complete: callable
         '''
+
         self._take_video(filename=filename, on_complete=on_complete)
 
     # private
 
     def _take_picture(self, **kwargs):
         raise NotImplementedError()
-        
+
     def _take_video(self, **kwargs):
         raise NotImplementedError()
 
 
+class Contacts(object):
+    '''Contacts Facade
+
+    .. versionadded:: 1.2.4
+
+    '''
+
+    _contacts = []
+
+    def __init__(self):
+        """Using refresh that should full fil _contacts"""
+        self.refresh()
```

Comment:
You missed to call the object constructor as well: `super(Contacts, self).__init__()`